web_app/model_view.py

```diff
@@ -2,23 +2,63 @@
 import numpy as np
 import pandas as pd
 import streamlit as st
-
-from models import AC_model1, AC_model2, AC_model3, AC_model4, AC_model5, AC_model6, IC_model1
+import json
+from models import AC_model1, AC_model2, AC_model3, AC_model4, AC_model5, AC_model6, IC_model1, IC_model2, IC_model3, IC_model4
```

Comment:
Variables should never be named by numbers except for indices in long equations. Please name them by there publication identifier so for example IC_model_Soares1999

Reply:
done